### PR TITLE
Virus-Hemo Infections

### PR DESCRIPTION
**Pull request by Icarus-The-Sun**
## About The Pull Request

Removes a single line to make hemos infectable.

## Changelog

:cl:
del: Removes Hemophage's Virus Immunity.
/:cl:

### CODE REVIEW

**Comment by Icarus-The-Sun** [comment]
The map tests fail for reasons beyond my understanding, screenshot test fails because I altered hemo code.

File: modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm [hunk 1]
```diff
@@ -312,8 +311,7 @@
 			SPECIES_PERK_TYPE = SPECIES_POSITIVE_PERK,
 			SPECIES_PERK_ICON = "biohazard",
 			SPECIES_PERK_NAME = "Viral Symbiosis",
-			SPECIES_PERK_DESC = "Hemophages, due to their condition, cannot get infected by \
-								other viruses and don't actually require an external source of oxygen \
+			SPECIES_PERK_DESC = "Hemophages, due to their condition don't actually require an external source of oxygen \
```
Comment (nevimer) [inline modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm:314]:
```suggestion
			SPECIES_PERK_DESC = "Hemophages, due to their condition don't actually require an external source of oxygen \ // BUBBER EDIT
```

File: modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm [hunk 2]
```diff
@@ -312,8 +311,7 @@
 			SPECIES_PERK_TYPE = SPECIES_POSITIVE_PERK,
 			SPECIES_PERK_ICON = "biohazard",
 			SPECIES_PERK_NAME = "Viral Symbiosis",
-			SPECIES_PERK_DESC = "Hemophages, due to their condition, cannot get infected by \
-								other viruses and don't actually require an external source of oxygen \
+			SPECIES_PERK_DESC = "Hemophages, due to their condition don't actually require an external source of oxygen \ // BUBBER EDIT
```
Comment (nevimer) [inline modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm:314]:
```suggestion
			SPECIES_PERK_DESC = "Hemophages, due to their condition don't actually require an external source of oxygen \
```

File: modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm [hunk 3]
```diff
@@ -312,8 +311,7 @@
...
 								to stay alive.",
```
Comment (nevimer) [inline modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm:315]:
```suggestion
								to stay alive.", // BUBBER EDIT
```

**Review by nevimer — APPROVED** [review]

**Pull request merged**